huaweicloud/services/acceptance/acceptance.go

```diff
@@ -1417,6 +1419,13 @@ func TestAccPreCheckImsBackupId(t *testing.T) {
 	}
 }
 
+// lintignore:AT003
+func TestAccPreCheckImsImageUrl(t *testing.T) {
+	if HW_IMS_IMAGE_URL == "" {
+		t.Skip("HW_IMS_IMAGE_URL must be set for IMS data image with OBS bucket")
```

Comment:
In order to improve the utilization of variables, it is recommended to modify it to
```
t.Skip("HW_IMS_IMAGE_URL must be set for this acceptance test")
```

Add a comment where this method is referenced.